zebra-state/src/service/finalized_state.rs

```diff
@@ -39,33 +39,21 @@ pub struct FinalizedState {
     queued_by_prev_hash: HashMap<block::Hash, QueuedBlock>,
     max_queued_height: i64,
 
-    hash_by_height: sled::Tree,
-    height_by_hash: sled::Tree,
-    block_by_height: sled::Tree,
-    tx_by_hash: sled::Tree,
-    utxo_by_outpoint: sled::Tree,
-    sprout_nullifiers: sled::Tree,
-    sapling_nullifiers: sled::Tree,
-    // sprout_anchors: sled::Tree,
-    // sapling_anchors: sled::Tree,
```

Comment:
Should we have a comment where the ColumnFamilies are defined indicating that these are to be added?

Reply:
Not sure if that's terribly important